ckks/ciphertext.go

```diff
@@ -45,7 +45,7 @@ func NewCiphertextAtLevelFromPoly(level int, poly [2]*ring.Poly) *Ciphertext {
 	v0, v1 := new(ring.Poly), new(ring.Poly)
 	v0.IsNTT, v1.IsNTT = true, true
 	v0.Coeffs, v1.Coeffs = poly[0].Coeffs[:level+1], poly[1].Coeffs[:level+1]
-	return &Ciphertext{Ciphertext:&rlwe.Ciphertext{Value:[]*ring.Poly{v0, v1}}, Scale:0}
+	return &Ciphertext{Ciphertext: &rlwe.Ciphertext{Value: []*ring.Poly{v0, v1}}, Scale: 0}
```

Comment:
Godoc must be explicit that the returned ciphertext shares its backing coefficient arrays with `Poly`s in `poly`.